src/registry/routes/helpers/get-component.js

```diff
@@ -149,7 +150,12 @@ module.exports = function(conf, repository){
           renderMode: renderMode
         });
 
-        if(isUnrendered){
+        if (skipRendering) {
+          callback({
+            status: 200,
+            response: response
```

Comment:
Also, the problem of doing this here (instead of around line 78) is that here we go through validation and data closure execution, which I believe is not needed for the info purposes. I think we don't want to just skip rendering, but skip execution at all.